ovs/lib/update.py

```diff
@@ -89,7 +91,9 @@ def _get_package_information_core(client, package_info):
             binaries = PackageManager.get_binary_versions(client=client, package_names=UpdateController._packages_core['storagedriver'])
             installed = PackageManager.get_installed_versions(client=client, package_names=UpdateController.packages_core_all)
             candidate = PackageManager.get_candidate_versions(client=client, package_names=UpdateController.packages_core_all)
-            if set(installed.keys()) != set(UpdateController.packages_core_all) or set(candidate.keys()) != set(UpdateController.packages_core_all):
+            installed_difference = set(UpdateController.packages_core_all) - set(installed.keys())
+            candidate_difference = set(UpdateController.packages_core_all) - set(candidate.keys())
+            if len(installed_difference | candidate_difference) > 3 or any(['volumedriver' not in package and 'alba' not in package for package in installed_difference | candidate_difference]):
```

Comment:
Shouldn't we check here, that as soon as there's a difference, its either all `non-ee` packages or all `ee` packages
Cause with this check, its possible to have a mixture of packages between `ee` and `non-ee` and that's not allowed right?

Same remark in the ASD-manager code then actually